contrib/slurm-test/slurm_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+docker-compose up -d
+docker ps
+docker cp toil_workflow.py slurm-test_slurmmaster_1:/home/admin
+#Assumes sort.py copied to this directory during gitlab run for ease of access
+docker cp sort.py slurm-test_slurmmaster_1:/home/admin
+docker cp fileToSort.txt slurm-test_slurmmaster_1:/home/admin
+docker cp toil_workflow.py slurm-test_slurmmaster_1:/home/admin
+GIT_COMMIT=$(git rev-parse HEAD)
+docker exec slurm-test_slurmmaster_1 sudo apt install python3-pip -y
+docker exec slurm-test_slurmmaster_1 pip3 install "git+https://github.com/DataBiosphere/toil.git@${GIT_COMMIT}"
+docker exec slurm-test_slurmmaster_1 sinfo -N -l
+docker exec slurm-test_slurmmaster_1 python3 /home/admin/toil_workflow.py file:my-job-store --batchSystem slurm --disableCaching --retryCount 0
+docker cp slurm-test_slurmmaster_1:/home/admin/output.txt output_Docker.txt
+docker exec slurm-test_slurmmaster_1 python3 /home/admin/sort.py file:my-job-store --batchSystem slurm --disableCaching --retryCount 0
+docker cp slurm-test_slurmmaster_1:/home/admin/sortedFile.txt sortedFile.txt
+docker-compose stop
+./check_out.sh
+rm sort.py
```

Comment:
Having the Gitlab script copy sort.py into here, and then deleting it, makes the setup very tightly coupled: you can *only* run this script as part of the Gitlab script. If you run it on its own, manually, it will complain that sort.py isn't there. If you provide it, it will run once, but delete it, so you can't run it again to test something else without providing a new copy of sort.py.

If the test fails, sort.py sticks around, untracked and un-cleaned-up.

I would recommend symlinking sort.py into here from its normal location, so there can be just one copy, but so that it will also be available here. I am pretty sure `docker cp` will follow the symlink to get the real sort.py. Then you won't have to deal with copying the file around, or with wanting to clean up afterward.